src/ert/_c_wrappers/enkf/enkf_main.py

```diff
@@ -378,8 +370,7 @@ def addDataKW(self, key: str, value: str) -> None:
         # self.resConfig().substitution_list is still
         # used by workflows to do substitution. For now, we
         # need to update this here.
```

Comment:
This comment :arrow_up:  seems a bit outdated as-is now and should maybe be removed?